torchvision/transforms/functional_tensor.py

```diff
@@ -132,7 +132,7 @@ def hflip(img: Tensor) -> Tensor:
 def crop(img: Tensor, top: int, left: int, height: int, width: int) -> Tensor:
     _assert_image_tensor(img)
 
-    _, h, w = get_dimensions(img)
+    w, h = get_image_size(img)
```

Comment:
I found this while debugging `get_dimensions`. Since the number of channels is not used, we can get away with using `get_image_size`. Took me a while to figure out that it returns width first :roll_eyes: 

LMK if you don't want this change in this PR.

Reply:
I will be removing the use of `get_image_size()` from stable `ElasticTransform` on a separate PR.